nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -201,6 +201,8 @@
 
 - [Glances](https://github.com/nicolargo/glances), an open-source system cross-platform monitoring tool. Available as [services.glances](option.html#opt-services.glances).
 
+- [git-worktree-switcher](https://github.com/mateusauler/git-worktree-switcher), switch between git worktrees with speed. Available as [programs.git-worktree-switcher](#opt-programs.git-worktree-switcher.enable)
```

Comment:
24.11 has been release by now, could you please move this to the 25.05 release notes